### ISSUE TITLE
Use position sticky instead of JavaScript

### ISSUE THREAD

**Issue by nickcolley** [alphagov/static#886]
https://github.com/alphagov/static/blob/9e8795b262c948a65b798029864cbe04f193262a/app/assets/javascripts/modules/sticky-element-container.js

Current browser usage is too low since Chrome has only just shipped it (http://caniuse.com/#feat=css-sticky), however once people switch over it might be enough users to consider this an enhancement we shouldnt polyfill and only use as an enhancement.

**Comment by dsingleton** [comment]
Copying my comments from slack.

My answer would be: it depends. What issues we have with the current implementation (perf, complexity, too much js?), how many folks would be be affected by the change? (html pubs tend to have a different UA distribution that mainstream content)

Also I'd suggest i’d look at analytics and see how widely used the feature is - that would likely help decide, but might require adding some analytics instrumentation

From memory we debated that at the time - could we just remove the feature, was it being used? In the interest of keeping migration moving we added something with parity and deferred the decision.

So I think the better question is, is this useful, if not, can we remove it? Or, if it’s not widely used, then a progressive approach that doesn’t cover all users is easier to make the case for.
